packages/suite/src/components/suite/modals/CancelCoinjoin.tsx

```diff
@@ -29,11 +29,13 @@ interface CancelCoinjoinProps {
 }
 
 export const CancelCoinjoin = ({ onClose }: CancelCoinjoinProps) => {
-    const { account } = useSelector(state => state.wallet.selectedAccount) as SelectedAccountLoaded;
+    const account = useSelector(selectSelectedAccount);
 
-    const { stopCoinjoinSession } = useActions({
-        stopCoinjoinSession: coinjoinAccountActions.stopCoinjoinSession,
-    });
+    const dispatch = useDispatch();
+
+    if (!account) {
+        return;
```

Comment:
return null instead of undefined to be usable as a JSX component in UserContextModal.tsx
see https://github.com/trezor/trezor-suite/actions/runs/3480766521/jobs/5820987582#step:13:203
```suggestion
        return null;
```

Reply:
Interesting TS still throws, when React 18 is supposed to [not throw](https://github.com/facebook/react/pull/21869) :D

Reply:
76c7b1d6d177798b40a8c34a8fb43461eb3aec34

Reply:
this is not because of React types but our modal types...